defaults/main.yml

```diff
@@ -1,7 +1,24 @@
 ---
 ## General
+mysql_flavour: mysql
+# Alt option:
+# mysql_flavour: mariadb
 mysql_config_version: 0.8.6-1
 
+mysql_deb_package: "mysql-apt-config_{{ mysql_config_version }}_all.deb"
+mysql_deb_repo: "http://dev.mysql.com/get/{{ mysql_deb_package }}"
+
+mysql_required_libs:
+  - python-mysqldb
+
+mysql_packages:
+  - mysql-common
+  - mysql-server
+
+# Packages for MariaDB:
```

Comment:
I would prefer two files for these configuration values depending on MySQL flavours. 

Something like `vars/mysql.yml` and `vars/mariadb.yml` setting that values.

Reply:
Ok, changed.